test/unit/helper-specs.js

```diff
@@ -89,4 +89,11 @@ describe("Helpers", function () {
       logger.warn.called.should.equal(false);
     });
   });
-});
+  describe('xcode version', function () {
+    it('parses xcode version with space', function () {
+      var expected = "/Applications/ Xcode 6.1.1.app/Contents/Developer";
+      var  = "/Applications/\\ Xcode\\ 6.1.1.app/Contents/Developer"
```

Comment:
looks like the variable name is missing here @tarunsujit. fix this and i think we're good to! and the first line (line 94) should be `actual`, this line is `expected`.